features/security/updating.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Account editing
+  As a user
+  In order to increase my comfort
+  I want to not have to enter my password on simple changes
+
+
+  Scenario: simple changes
+    Given a user
+    And the user is logged in
+    When he views his account settings
+    And he enters the first name 'Frankenstein'
+    And the user submits the form
+    Then his first name should be 'Frankenstein'
+
+  Scenario: dangerous changes without password
+    Given a user
+    And the user is logged in
+    When he views his account settings
+    And he enters the email 'email@email.email'
+    And the user submits the form
+    Then his email should not have changed
+
+  Scenario: dangerous changes with password
```

Comment:
Are the changes really "dangerous"? Maybe "important" is a better word.

Reply:
I'd say: "dangerous" for the security of the account.